### ISSUE TITLE
html tags not working

### ISSUE THREAD

**Issue by jinhi** [puikinsh/shapely#155]
HTML tags are not rendering correctly. The just show up as code. In the third parallax widget at the site below:

http://69.89.31.185/~woodstu0/dojo/

you'll see that `<em>` and `<br>` are not working.

I've installed the beta as well as the companion plugin. Please help!

Jinhi

**Comment by jinhi** [comment]
I'm not sure if this is related, but in the old theme, I was able to have a google map on one of the widgets (the Contact widget). But here, it's just showing up as code.

**Comment by jinhi** [comment]
I wanted to update this ticket, since I see it was assigned to someone. 

Just today, I noticed in my WP dashboard that I needed to install the companion plugin. This was strange, since I had already installed the companion plugin using the download link that was provided on Shapely's site. Regardless, I deactivated and then deleted the companion plugin and then reinstalled/activated it using the link given in my dashboard. Since then, `<em>` and `<br>` are rendering correctly--though only on desktop. On mobile, the `<br>` is not recognized. I've attached a screengrab of the page below:

http://69.89.31.185/~woodstu0/dojo/
![trashme](https://cloud.githubusercontent.com/assets/25909536/23281498/899deda4-f9eb-11e6-959f-9bfaf3c509a4.gif)

I should note that the coding for the google map is still not working though. Just the raw code appears.
